src/platforms/unity/troubleshooting.mdx

```diff
@@ -12,7 +12,16 @@ The release 0.6.1 is missing the native libraries needed to build for iOS. This
 
 ### Package Manager - `Cannot perform upm operation: Unable to add package`
 
-Confirm the URL doesn't have a trailing whitespace at the end. The Unity Package Manager will fail to find the package if a trailing whitespace is appended.
+If you encounter the following error:
+
+```
+[Package Manager Window] Unable to add package [https://github.com/getsentry/unity.git#0.9.1 ]:
+  Error when executing git command. fatal: invalid refspec '0.9.1 '
+```
+
+Confirm the URL doesn't have a trailing whitespace between the URL. The Unity Package Manager will fail to find the package if a trailing whitespace is appended.
```

Comment:
Between?

Reply:
In this case just "in" is correct rather than "between". You would use "between" if there are two things (e.g., spaces between the beginning and end of the URL).
 
```suggestion
Confirm the URL doesn't have a trailing whitespace or spaces in the URL. The Unity Package Manager will fail to find the package if a trailing whitespace is appended.
```